server/middleware/setUpWebSecurity.ts

```diff
@@ -41,6 +41,7 @@ export default function setUpWebSecurity(): Router {
             "'self'",
             config.apis.govukOneLogin.url,
             config.apis.govukOneLogin.url.replace('oidc', 'signin'),
+            config.pvbUrl,
```

Comment:
This is needed here because if PVB url is not in the CSP, browsers won't follow the redirect.